tests/test_cwl.py

```diff
@@ -5,7 +5,7 @@
 import yaml
 
 from config import TESTS_RESOURCES
-from fairworkflows import cwl
+from fairworkflows import cwl, cwl_cli
```

Comment:
Could you move the tests for `cwl_cli` to a different module?

Reply:
Done, and rebased to master now